### ISSUE TITLE
Oxipng AlphaOptim setting should be disabled by default

### ISSUE THREAD

**Issue by chgi** [GoogleChromeLabs/squoosh#1290]
**Describe the bug**
In [codecs/oxipng/src/lib.rs](https://github.com/GoogleChromeLabs/squoosh/blob/97d13de273b67ee01a80cb32161f1bb2103b73cb/codecs/oxipng/src/lib.rs), alpha optimization enabled. 
This breaks the lossless promise of PNG and actually makes a difference for certain use cases like packed textures.
Please see donmccurdy/glTF-Transform#704 for a real world example.

**Expected behavior**
I believe, alpha optimization should either disabled or (preferrably) configurable.

**Version:**
at least version 0.4.0 on [npm](https://www.npmjs.com/package/@squoosh/lib)

**Comment by jakearchibald** [comment]
I don't think it should be off by default, but making it configurable sounds good. Probably the same for lossless webp.
